### PR TITLE
[EDX-367] Trigger search via cmd+k keyboard shortcut

### PR DESCRIPTION
**Pull request by snikidev**
> _NOTE TO REVIEWERS_ - please **do not review PRs in the `DRAFT` state**, as the PR may change substantially before it is ready to review. Thanks.

## Description

Added CMD+K keys on the search bar + fixed mobile bugs

* [Jira ticket](https://ably.atlassian.net/browse/EDX-367).

### CODE REVIEW

**Comment by MymmiJ** [comment]
Make sure tests are passing!
Consider running `npm run repo-githooks` to have tests run before every commmit.

**Review by MymmiJ — APPROVED** [review]

File: src/components/Header/TopMainNav/HamburgerMenu/HamburgerMenu.module.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.hamburgerMenu {
+  @apply overflow-y-scroll;
+  max-height: calc(100vh - 245px);
```
Comment (aralovelace) [inline src/components/Header/TopMainNav/HamburgerMenu/HamburgerMenu.module.css:3]: minor but can we change the 245px to rem?

File: src/components/Header/TopMainNav/HamburgerMenu/HamburgerMenu.tsx [hunk 2]
```diff
@@ -1,12 +1,14 @@
 import React, { useState } from 'react';
+import cn from 'classnames';
```
Comment (aralovelace) [inline src/components/Header/TopMainNav/HamburgerMenu/HamburgerMenu.tsx:2]: i think `import cn from 'classnames';` is not being used in this file component anymore?

File: src/components/Header/TopMainNav/SearchBar/SearchBar.test.tsx [hunk 3]
```diff
@@ -19,7 +19,12 @@ describe('<SearchBar />', () => {
 
   it('should open search box on typing in search input and close on click outside', async () => {
     const user = userEvent.setup();
-    render(<SearchBar displayMode={DisplayMode.FULL_SCREEN} />);
+    render(
```
Comment (aralovelace) [inline src/components/Header/TopMainNav/SearchBar/SearchBar.test.tsx:22]: `fireEvent` is not being used here as well?

**Review by aralovelace — COMMENTED** [review]
Just small comment LGTM

**Pull request merged**